examples/seismic/acoustic/operators.py

```diff
@@ -10,10 +10,19 @@ def laplacian(field, m, s, kernel):
     order in time formulation, the 4th order time derivative is replaced by a
     double laplacian:
     H = (laplacian + s**2/12 laplacian(1/m*laplacian))
-    :param field: Symbolic TimeFunction object, solution to be computed
-    :param m: square slowness
-    :param s: symbol for the time-step
-    :return: H
+
+    Parameters
+    ----------
+    field : TimeFunction
+        The computed solution.
+    m : float
+        Square slowness.
+    s :
+        Time-step.
+
+    Returns
+    -------
+    H FIX
```

Comment:
reminder (@mloubout ); I think the actual description at the top of this docstring could be revisited

Reply:
Sure, not sure how to rephrase it though

Reply:
maybe the description is fine, but we can drop the "Returns" paragraph as it would be redundant?

Reply:
Drop the Returns, then

Reply:
It is still here

Reply:
?! ok, dropped

Reply:
dropped